label_studio_ml/examples/tesseract/test_model.py

```diff
@@ -12,28 +15,66 @@ def client():
         yield client
 
 
-def test_basic_interactions(client):
+@pytest.fixture
+def model_dir_env(tmp_path, monkeypatch):
+    model_dir = tmp_path / "model_dir"
+    model_dir.mkdir()
+    monkeypatch.setattr(BBOXOCR, 'MODEL_DIR', str(model_dir))
+    return model_dir
```

Comment:
Added changes to avoid cache conflicts based on [this commit](https://github.com/HumanSignal/label-studio-ml-backend/commit/71a50c07886d295e60f5065f44136505fb05185e#diff-98a8b12bac16156c4f3cd9abf1b74f4f0094113c1d6da6874ea4fe7a1bc7108d).